src/main/java/org/jenkinsci/plugins/gitserver/CSRFExclusionImpl.java

```diff
@@ -40,48 +40,46 @@
  */
 @Extension
 public class CSRFExclusionImpl extends CrumbExclusion {
+    private static final String BOGUS = "bogus";
 
+    @Override
     public boolean process(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws IOException, ServletException {
         if (!"application/x-git-receive-pack-request".equals(request.getHeader("Content-Type")))
             return false;
 
-//        String path = request.getPathInfo();
-//        if(!path.contains("/repo.git/") || !path.endsWith("/git-receive-pack"))
-//            return false;
-
         HttpServletRequestWrapper w = new HttpServletRequestWrapper(request) {
             @Override
             public String getQueryString() {
-                return "bogus";
+                return BOGUS;
             }
 
             @Override
             public String getParameter(String name) {
-                return "bogus";
+                return BOGUS;
             }
 
             @Override
-            public Map getParameterMap() {
+            public Map<String, String[]> getParameterMap() {
                 return Collections.emptyMap();
             }
 
             @Override
-            public Enumeration getParameterNames() {
-                return new Vector().elements();
+            public Enumeration<String> getParameterNames() {
+                return Collections.emptyEnumeration();
             }
 
             @Override
             public String[] getParameterValues(String name) {
-                return new String[]{"bogus"};
+                return new String[]{BOGUS};
             }
 
             @Override
             public String getMethod() {
-                return "BOGUS";
+                return BOGUS.toUpperCase();
```

Comment:
Better
```suggestion
                return BOGUS.toUpperCase(Locale.ROOT);
```
though there is no `i` so it likely does not matter

Reply:
This is a good idea. Will do shortly.